ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-settings/map-settings.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import MapSettingsPresentation from '../../presentation/map-settings'
+import Dropdown from '../../presentation/dropdown'
+import { hot } from 'react-hot-loader'
+const Common = require('../../../js/Common.js')
+const user = require('../../../component/singletons/user-instance.js')
+const mtgeo = require('mt-geo')
+
+const exampleLat = '14.94',
+  exampleLon = '-11.875'
+const examples: { [index: string]: string } = {
+  degrees: `${mtgeo.toLat(exampleLat)} ${mtgeo.toLon(exampleLon)}`,
+  decimal: `${exampleLat} ${exampleLon}`,
+  mgrs: '4Q FL 23009 12331',
+  utm: '14 1925mE 1513mN',
+}
+
+const save = (newFormat: string) => {
+  Common.queueExecution(() => {
+    var preferences = user.get('user').get('preferences')
+    preferences.set({
+      coordinateFormat: newFormat,
+    })
+    preferences.savePreferences()
+  })
+}
+
+const Span = styled.span`
+  padding-right: 5px;
+`
+type State = {
+  selected: string
+}
+
+class MapSettings extends React.Component<{}, State> {
+  constructor(props: {}) {
+    super(props)
+    this.state = {
+      selected: user
+        .get('user')
+        .get('preferences')
+        .get('coordinateFormat'),
+    }
+  }
+
+  update(newFormat: string) {
+    save(newFormat)
+    this.setState({ selected: newFormat })
```

Comment:
An alternative to having to set this in both places (preferences and then setState) would be to use the backbone higher order component so we could listen for changes to the user's coordinateFormat preference.

https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/container/navigation-right-container/navigation-right-container.tsx#L16 is a good example of using it.